docs/reference/contributing/guidelines/workflow.md

```diff
@@ -221,9 +221,20 @@ We use many other labels to summarize the scope and effect of the changes.
 - *DO NOT MERGE* - This pull request contains changes that may be in a draft state and submitted purely for review, or may contain breaking changes that have not been considered.
 - *devices: 'name'* - The pull request specifically affects the named device(s).
 - *component: 'name'* - The pull request specifically affects the named component. Component names follow the structure of Mbed OS (`ble`, `storage`, `tls` and so on).
+- *rollup PR* - This pull request is ready to begin CI testing, but will be tested with other pull requests at once.
 
 The following labels summarize the scope of the pull request.
 
 - *scope: bug-fix*.
 - *scope: feature*.
 - *scope: new-target*.
+
+#### Rollup Pull Requests
+
+At times, Mbed OS will contain many small pull requests that are either unrelated to each other (completely different tests or targets), or have a low risk of failing CI testing (docs changes). At the discretion of a maintainer, a pull request will be additionally tagged with the *rollup PR* label and a script will create a rollup pull request to take through testing. Upon success, the rollup pull request is merged as normal, and all pull requests that were marked with the *rollup PR* label will be marked as merged.
+
+No additional work from a pull request author is needed. If the rollup pull request fails testing, maintainers will root cause the issue, and reach out to the author(s) of the problematic pull request(s) that a problem was identified. Optionally, a maintainer may choose to regenerate the rollup pull request without the problem pull request(s).
+
+##### How it works
+ 
+Rollup pull requests are generated by creating a branch off of the HEAD of master. Once the branch is created, pull requests are scanned for the *rollup PR* label, and each pull request that is found to have the label, the same commands that would be used to merge a pull request into master are instead performed into the rollup branch. A pull request is then opened normally against master using this new branch.
```

Comment:
this is only explains how the rollup branch is created, not what happens later (testing / merging the changes).
It might be nice to add a bit of details on how the merge part of it works (PRs will appear as though they were merged normally in the order they were added to the rollup IIRC).

Reply:
each pull request ~that is found to have~ with the label

Reply:
> and each pull request that is found to have the label, the same commands that would be used to merge a pull request into master are instead performed into the rollup branch.

There's non-parallel construction here.

Reply:
I will add more, but I wanted to avoid adding _too_ much detail, less the purpose of the page get diluted.

Will work on my sentence structure(s).

Reply:
I'll add more. I was concerned about adding too much details in what I view is an overview page.

Reply:
:+1